test/unit/fixture/autocomplete_boundary_country.js

```diff
@@ -51,12 +51,12 @@ module.exports = {
         }
       }],
       'filter': [{
-        'match': {
-          'parent.country_a.ngram': {
+          'multi_match': {
+            'type': 'best_fields',
```

Comment:
`best_fields` is the default strategy, we might consider omitting the value completely?

Reply:
Agreed, this would be a good follow up change in `pelias/query`. Just saves a few characters in every elasticsearch query :)